### ISSUE TITLE
Merge two layers, But backprop only through one

### ISSUE THREAD

**Issue by superhans** [fchollet/keras#560]
Let's say I have code to merge two layers, like this : 
```
left = Sequential()
left.add(Dense(784, 50))
left.add(Dense(50, 50))
left.add(Activation('relu'))

right = Sequential()
right.add(Dense(784, 50))
right.add(Dense(50, 50))
right.add(Activation('relu'))

model = Sequential()
model.add(Merge([left, right], mode='concat'))

model.add(Dense(100, 10))
model.add(Activation('softmax'))
```

Now, it's apparent that concat(left,right) is used to generate the softmax output. But while training, I want to network to treat "left" as a constant , and not backprop through it, while "right" can be "backproped" through. Is this possible ?

**Comment by fchollet** [comment]
Just record the output of `left` (in test mode, i.e. using `.predict()`) then use this as the input to your 2-branch model. Do not include into your model layers that you don't intend on training.

**Comment by liyi193328** [comment]
I want to do the same experiment as @superhans , but I'm not quite understand  your ideas @fchollet   Can you explain it with more specific details. 
Thanks.

**Comment by farizrahman4u** [comment]
```python
right.trainable = False
```
@fchollet Wouldn't this work?

**Comment by liyi193328** [comment]
@farizrahman4u Thanks very much.
